src/resources/spi_linux.cc

```diff
@@ -46,21 +46,11 @@ PRIMITIVE(open) {
     close(fd);
     return Primitive::os_error(errno, process);
   }
-  result = ioctl(fd, SPI_IOC_RD_MAX_SPEED_HZ, &frequency);
```

Comment:
Not entirely clear why these two RD settings went away. Maybe add a comment to explain why we don't mess with them?

Reply:
The `SPI_IOC_RD_MAX_SPEED_HZ` is not a setting, but a way to read the current MAX_SPEED_HZ (basically a setter).
added a comment:
```
// "WR"ite the max speed and mode.
```